tests/conftest.py

```diff
@@ -1,34 +1,41 @@
 import os
-import sys
-
 import pytest
+import edi_835_parser
 
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
 
-import edi_835_parser
+@pytest.fixture
```

Comment:
These diffs got a bit messy. I added three new fixtures to support access to the "test" directories:
- base_test_directory
- test_input_directory: stores input edi files for test cases
- test_output_directory: stores the "converted" csv files for test cases

Additional updates were made to integrate these fixtures with existing test cases.